docs/guides/server/hostname.adoc

```diff
@@ -48,7 +48,7 @@ The result of this configuration is that you can continue to access {project_nam
 
 == Using a reverse proxy
 
-When a proxy is in use, the `proxy-headers` should be set. Depending on the hostname settings, some or all of the URL, may be dynamically determined.
+When a proxy is in use for edge, reencrypt, or http passthrough, the `proxy-headers` should be set. Depending on the hostname settings, some or all of the URL, may be dynamically determined.
```

Comment:
```suggestion
When a proxy is in use for edge, reencrypt, or https passthrough, the `proxy-headers` should be set. Depending on the hostname settings, some or all of the URL, may be dynamically determined.
```

Reply:
I do mean http, not https, here. But calling it http passthrough is probably confusing.